extensions/amazon-aws/src/ec2.tsx

```diff
@@ -91,8 +107,7 @@ function InstanceListItem(props: { instance: AWS.EC2.Instance }) {
               instance.InstanceId
             }
           />
-          <CopyToClipboardAction title="Copy Public IP" content={instance.PublicIpAddress || ""} />
-          <CopyToClipboardAction title="Copy Private IP" content={instance.PrivateIpAddress || ""} />
+          {ipActions}
```

Comment:
here you can use mechanism like this, instead of using array:

```ts
<CopyToClipboardAction key="privateIpAction" title="Copy Private IP" content={instance.PrivateIpAddress || ""} />
{instance.PublicIpAddress && <CopyToClipboardAction key="publicIpAction" title="Copy Public IP" content={instance.PublicIpAddress} />}
```

Reply:
Thanks a lot for your tip